AcumaticaSurveyPkg/Customization/_project/ProjectMetadata.xml

```diff
@@ -1,2 +1,2 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<project name="AcumaticaSurveyPkg" level="0" description="" />
\ No newline at end of file
+<project name="Customization" level="0" description="" />
```

Comment:
@markoan we need to keep AcumaticaSurveyPkg as project name

Reply:
Done in latest commit.